### DISCUSSION TITLE
Feature suggestion: Custom lists

### DISCUSSION THREAD

**Discussion by ToxicNaPalm** [civitai/civitai#461] [Ideas]
I think it would be beneficial if beyond just liking models, I could create lists that would be either public or private. An example would be if I stumbled across a model with western comic style, it would have a link to browse any public lists other users saved that model to and I could quickly discover more models also about comics. This would greatly enable and encourage content curation on the site.

**Comment by JustMaier** [comment]
I like this idea. Seems like it could help with discoverability as well. I think we'd need to add a place on the user page where you could see public lists that user has made.

**Comment by kuso-ge** [comment]
This sounds nice, there's too much junk and it's getting hard keeping track of which checkpoint and Lora's I use and I like. I only use the Like button so that it wouldn't get buried among the thousands of junk floating around everyday, but with the amount of checkpoints I liked it's very hard to keep track my favorites among them. Custom lists would also allow users make specific group for each type of Lora, checkpoint, they want. (i.e. Anime style checkpoints, Realistic checkpoints, character Lora's, Style Lora's, Landscape style checkpoints, etc...).

Edit:
One of the best of use of these is also keeping track of your favorite Lora's and its triggerwords and for any updates for it. Since there's like duplicate Lora's, some better than others. For example, Tatsumaki has like 6 Lora's and sometimes I forget which one I use, using the Like button has lost its value to me because of the amount I liked just for tracking purposes.

**Comment by namelessentity** [comment]
I absolutely would love this because currently I use my like button as a 'to download later' button as most of the time I'm on a metered laptop connection when I browse and only download on my desktop.
